app/Observers/EntityFileObserver.php

```diff
@@ -27,7 +27,7 @@ public function saving(EntityFile $entityFile)
      */
     public function saved(EntityFile $entityFile)
     {
-        // When adding or changing an entity note to an entity, we want to update the
+        // When adding or changing an post to an entity, we want to update the
```

Comment:
changed entity_note to post, but it's also wrong, should be "file" in both cases here :)